cpp/ql/src/Likely Bugs/Likely Typos/FutileParams.qhelp

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>A function is called with arguments despite having an empty parameter list. This may indicate
+that the incorrect function is being called, or that the author misunderstood the function.</p>
```

Comment:
Please add text to explain the difference between `()` in C, `(void)` in C/C++, and `()` in C++.